contributing.md

```diff
@@ -24,6 +24,29 @@ Whether you are a novice or experienced software developer, all contributions an
   make installdeps
   git checkout -b issue####-branch_name
   ```
+* You will need to install Spark & Scala to run all unit tests:
+
+     **macOS** (use [Homebrew](https://brew.sh/)):
+     ```bash
+     brew tap AdoptOpenJDK/openjdk
+     brew install --cask adoptopenjdk11
+     brew install scala
+     brew install apache-spark
+     ```
+
+     **Ubuntu**:
+     ```bash
+     sudo apt install openjdk-11-jre openjdk-11-jdk scala -y
+     ```
+     
+     **Amazon Linux**:
+     ```bash
+     sudo amazon-linux-extras install java-openjdk11 scala -y
+     ```
+
+In order to use Woodwork with Dask or Koalas DataFrames, the following commands must be run for your library of choice prior to installing Woodwork with conda: conda install dask for Dask or conda install koalas and conda install pyspark for Koalas.
```

Comment:
Can the commands here be put in back-ticks?

Reply:
Didn't mean to include that. The `make installdeps` above should take care of this. 
Removed.